.goreleaser.yml

```diff
@@ -53,6 +53,9 @@ archives:
       - src: release_deps/VERSION.txt
         dst: "."
         strip_parent: true
+      - src: release_deps/com.observiq.collector.plist
+        dst: "install"
+        strip_parent: true
```

Comment:
Is there a guide or read me that goes over the different fields in this file, it seems pretty straight forward in regards to knowing that this is talking about the archives and what not, but am unsure what all other fields are included with this. Basically wondering if there is a readme like there is for otel operators.

Reply:
This specific section (archive) is here: https://goreleaser.com/customization/archive/
You should be able to find all the docs on that site.
LMK if you have some specific questions on any of these fields.